tests/test_end_to_end.py

```diff
@@ -296,3 +338,15 @@ def test_builtin_plugins(tmp_path):
 def test_missing_object_exception():
     invalid_page_id = "11111111111111111111111111111111"
     assert run_n2y([invalid_page_id]) != 0
+
+
+def test_links_to_pages():
```

Comment:
I thought, per the comments in the previous review, that we were going to remove `test_links_to_page`, especially given the comment you have here.

Here's my previous comment about this: https://github.com/innolitics/n2y/pull/69#discussion_r967368234

Please delete this test.